### PR TITLE
Detail pages, names and dates

### LINKED ISSUE #92: Display author name & date published on blogpost
Display author name on blogpost

### PR DESCRIPTION
**Pull request by onurakpolat**
fixes #92 and #94

Two goals:

- [x] Add author names and dates
- [x] Create details pages for blog posts

### CODE REVIEW

**Comment by onurakpolat** [comment]
fixes #92  and #94

**Comment by onurakpolat** [comment]
@juliendonck 

- [ ] Please check design of blog overview page and adjust if required
- [ ] Fix design of blog detail page (currently moves under top)

File: themes/rad/layouts/partials/blog-main.html [hunk 1]
```diff
@@ -5,7 +5,10 @@ <h1>Radicle Blog</h1>
 
   {{ range .Pages }}
   <article>
-    {{ .Content }}
+  	<a href="{{ .RelPermalink }}"><h1>{{ .Title }}</h1></a>
+  	<p id="author">{{ .Params.author }}, {{ dateFormat "2006-01-02" .Params.date }}</p>
+    {{ .Summary }}
+    <a href='{{ .Permalink }}'><nobr>Read more →</nobr></a>
```
Comment (xla) [inline themes/rad/layouts/partials/blog-main.html]: What does the `nobr` do for us here?

File: themes/rad/layouts/partials/blog-main.html [hunk 2]
```diff
@@ -5,7 +5,20 @@ <h1>Radicle Blog</h1>
...
+  	<div id="summary">
+  		{{ if .Params.summary }}
+		{{ .Params.summary | markdownify }}
+		{{ else }}
+		{{ .Summary }}
+		{{ end}}
+  	</div>
+  	{{ if .Truncated }}
+      <div>
+        <a href='{{ .RelPermalink }}'><nobr>Read more →</nobr></a>
```
Comment (xla) [inline themes/rad/layouts/partials/blog-main.html]: What does `nobr` do for us here?

**Comment by juliendonck** [comment]
Ok, added the styling. I think we need to take a closer look at the two summaries. @hxrts would you mind taking a look? They are the snippets shown on the blog page before you go into the individual blog post. Right now I feel they are kinda saying the same thing.

File: content/blog/radicle-architecture.md [hunk 3]
```diff
@@ -1,15 +1,14 @@
 ---
 
-title: "Radicle"
+title: "Radicle Architecture"
 date: 2019-03-28
 markup: mmark
 author: "Julian Arni and James Haydon"
 draft: false
+summary: "Radicle is a system for code collaboration that has several advantages over
```
Comment (jameshaydon) [inline content/blog/radicle-architecture.md]: How about: "In this post we explain the P2P architecture underlying Radicle: decentralised state machines stored on IPFS."

@jkarni

**Comment by juliendonck** [comment]
@onurakpolat You think we can have `previous post` and `next post` on the individual blog pages?

**Comment by juliendonck** [comment]
Was able to add it myself @onurakpolat. Could've probably tried myself first before asking 😐

File: themes/rad/layouts/partials/blog-main.html (hunk 1 above)
Comment (onurakpolat) [reply themes/rad/layouts/partials/blog-main.html]: was unnecessary -> removed.

File: themes/rad/layouts/partials/blog-main.html (hunk 2 above)
Comment (onurakpolat) [reply themes/rad/layouts/partials/blog-main.html]: was unnecessary -> removed.

File: content/blog/radicle-architecture.md (hunk 3 above)
Comment (onurakpolat) [reply content/blog/radicle-architecture.md]: @juliendonck I also wouldn't mind just having the first part repeat itself and not call it summary but sneap peak.A distinct summary could even be better I agree.

**Comment by juliendonck** [comment]
We should wait to merge this PR since it will break the link to the blogpost on hackernews. So let's let the Radicle team decide when it's a good time to merge this in. @jkarni you have the controls :)

**Comment by juliendonck** [comment]
Actually just realized that James’ suggestion has not been merged in. Will take a look at this this afternoon.

File: content/blog/radicle-architecture.md (hunk 3 above)
Comment (jkarni) [reply content/blog/radicle-architecture.md]: I agree with @jameshaydon's summary.

**Pull request merged**